patches/@react-native+assets-registry+0.72.0.patch

```diff
@@ -0,0 +1,52 @@
+diff --git a/node_modules/@react-native/assets-registry/registry.js b/node_modules/@react-native/assets-registry/registry.js
```

Comment:
Can you please add an explanation and link to the upstream change related to the additional of this patch to the PR description?

Reply:
The issue fixed by this patch no longer occurs, so I've removed this file